demo/api/src/email-campaign/email-campaign-content-scope.ts

```diff
@@ -15,4 +15,9 @@ export class EmailCampaignContentScope {
     @Field()
     @IsString()
     language: string;
+
+    @Property({ columnType: "text" })
+    @Field()
+    @IsString()
+    test: string;
```

Comment:
We should remove the test again, since it adds no value to the demo.

Reply:
87d88415598aaa5a8682558906b7e37b972850aa